doc/api/fs.md

```diff
@@ -1447,6 +1453,9 @@ The listener callback gets two arguments `(event, filename)`.  `event` is either
 `'rename'` or `'change'`, and `filename` is the name of the file which triggered
 the event.
 
+Please note the listener callback is attached to the `'change'` event
```

Comment:
I think the actual confusion in the original issue, is because of the previous paragraph. IINW, OP misunderstood the type of event (`rename` and `change`) as the actual events emitted. Perhaps we can change/explain the parameter `event`.

Thoughts @Trott?